### ISSUE TITLE
For song vc what should I do

### ISSUE THREAD

**Issue by panxin801** [yl4579/StarGANv2-VC#61]
Hello and thank you sharing your great work, but I have some questions.
1. For song vc with Madarian, I tried train a new starganv2vc model with pretrained ASR and F0 model, but the result sound not well, do you have some advice ?
2.  In song vc with Madarian, do i need to retrain a ASR or F0 model ?
I'm looking forward for your reply, and thank you again.

**Comment by Francis-Komizu** [comment]
> Hello and thank you sharing your great work, but I have some questions.
> 
> 1. For song vc with Madarian, I tried train a new starganv2vc model with pretrained ASR and F0 model, but the result sound not well, do you have some advice ?
> 2. In song vc with Madarian, do i need to retrain a ASR or F0 model ?
>    I'm looking forward for your reply, and thank you again.

Hello, panxin! I'm also working on singing vc with StarGANv2-VC. I didn't retrained F0 and ASR model. Instead, I made a dataset consisting of Mandarin songs, Mandarin, Japanese and English speech.  This is my [result](https://www.bilibili.com/video/BV11T411M7vR?spm_id_from=333.999.0.0&vd_source=017d04cd0542838d46958fe808b04d97).

**Comment by panxin801** [comment]
@Francis-Komizu   well, thank you for your reply, indeed I think starganvc using for song vc may need some further works to work out

**Comment by yl4579** [comment]
@panxin801  I'm currently working on singing conversion using this model with some further modifications for better performance. I may submit my work to INTERSPEECH next year.

**Comment by panxin801** [comment]
@yl4579  Well, Congratulations. I'm looking forward for your works .

**Comment by mraj96** [comment]
> @panxin801 I'm currently working on singing conversion using this model with some further modifications for better performance. I may submit my work to INTERSPEECH next year.

@yl4579, is INTERSPEECH 2022 September? If yes, can you share the paper link here

**Comment by yl4579** [comment]
@mraj96 Sorry, I mean INTERSPEECH next year so it'll be 2023.

**Comment by mayank-git-hub** [comment]
The main modification which makes StarGANv2-VC work on singing voice is the removal of pitch features from the instance normalization layers of the generator and having an absolute pitch reconstruction loss instead of a normalized pitch reconstruction loss.

**Comment by billnye2** [comment]
@mayank-git-hub Do you have a github for ROSVC? Couldn't find the source code, very interested!
